biodatasets/mednli/mednli.py

```diff
@@ -0,0 +1,176 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+State of the art models using deep neural networks have become very good in learning an accurate
+mapping from inputs to outputs. However, they still lack generalization capabilities in conditions
+that differ from the ones encountered during training. This is even more challenging in specialized,
+and knowledge intensive domains, where training data is limited. To address this gap, we introduce
+MedNLI - a dataset annotated by doctors, performing a natural language inference task (NLI),
+grounded in the medical history of patients. As the source of premise sentences, we used the
+MIMIC-III. More specifically, to minimize the risks to patient privacy, we worked with clinical
+notes corresponding to the deceased patients. The clinicians in our team suggested the Past Medical
+History to be the most informative section of a clinical note, from which useful inferences can be
+drawn about the patient.
+"""
```

Comment:
please add some information to the docstring concerning the form that the input should take (e.g. https://github.com/bigscience-workshop/biomedical/blob/master/biodatasets/n2c2_2006_deid/n2c2_2006_deid.py#L34)